src/sugar3/speech.py

```diff
@@ -278,7 +278,12 @@ def make_pipeline(self, command):
             self.stop_sound_device()
             del self._pipeline
 
-        self._pipeline = Gst.parse_launch(command)
+        try:
+            self._pipeline = Gst.parse_launch(command)
+        except:
```

Comment:
​
I haven't installed espeak module on my ubuntu 15.04, that is what I got ^^

Ignacio Rodríguez
nachoel01@gmail.com
SugarLabs at Facebook
<https://www.facebook.com/pages/SugarLabs/187845102582>

2015-01-05 15:04 GMT-02:00 Gonzalo Odiard <notifications@github.com>:

> In src/sugar3/speech.py
> <https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/184#discussion-diff-22473420>
> :
>
> > @@ -278,7 +278,12 @@ def make_pipeline(self, command):
> >              self.stop_sound_device()
> >              del self._pipeline
> >
> > -        self._pipeline = Gst.parse_launch(command)
> > +        try:
> > +            self._pipeline = Gst.parse_launch(command)
> > +        except:
>
> Not sure if you will get the same error by rename than by uninstall.
> Please check if you can.
> … <#14abb0eba8c8dda1_>
> On Mon, Jan 5, 2015 at 1:53 PM, Ignacio Rodríguez <
> notifications@github.com> wrote: In src/sugar3/speech.py <
> https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/184#discussion-diff-22472585>
> : > @@ -278,7 +278,12 @@ def make_pipeline(self, command): >
> self.stop_sound_device() > del self._pipeline > > - self._pipeline =
> Gst.parse_launch(command) > + try: > + self._pipeline =
> Gst.parse_launch(command) > + except: @godiard
> <https://github.com/godiard> <https://github.com/godiard> instead of
> uninstall, I tried to change the plugin name, to random thing, the error I
> got is GError. btw, working on the Gst.Init patch. -- Reply to this email
> directly or view it on GitHub <
> https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/184/files#r22472585>
> .
>  -- Gonzalo Odiard
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/184/files#r22473420>
> .
>